lib/modules/manager/terraform/extractors/resources/helm-release.ts

```diff
@@ -27,9 +32,14 @@ export class HelmReleaseExtractor extends DependencyExtractor {
         };
         if (!helmRelease.chart) {
           dep.skipReason = 'invalid-name';
+        } else if (isOCIRegistry(helmRelease.chart)) {
+          // For oci charts, we remove the oci:// and use the docker datasource
+          dep.depName = helmRelease.chart.replace(ociRegex, '');
```

Comment:
```suggestion
          dep.depName = helmRelease.chart.replace('oci://', '');
```

Can only be once and on start inside this if condition. Otherwise the chart will be invalid anyways.